src/pages/home/home.ts

```diff
@@ -64,4 +65,8 @@ export class HomePage {
   public goToAddView(): void {
     this.navCtrl.push(AddPage);
   }
+
+  goToWalletDetails(wallet: any) {
+    this.navCtrl.push(WalletDetailsPage, {wallet: wallet});
```

Comment:
`wallet` param should be replaced by a single element, not an object. Ej, `walletId`

Reply:
+1